changelogs/fragments/2204-github_repo-fix-baseurl_port.yml

```
@@ -0,0 +1,2 @@
+bugfixes:
+  - PyGithub bug does not allow explicit port in base_url for github_repo.py (https://github.com/PyGithub/PyGithub/issues/1913). Specifying port is not required.
```

hi @atorrescogollo  there is a format to conform to. Each line we write in there becomes a line in the changelog, like:
https://github.com/ansible-collections/community.general/blob/stable-2/CHANGELOG.rst#bugfixes

```suggestion
  - github_repo - PyGithub bug does not allow explicit port in ``base_url``. Specifying port is not required. (https://github.com/PyGithub/PyGithub/issues/1913). 
```

---

(Sorry I didn't provide all the information you needed the first time)

---

Ah okay, I see :)